src/DocBlock/Annotation.php

```diff
@@ -157,7 +157,7 @@ public function getTypes()
      */
     public function setTypes(array $types)
     {
-        $pattern = '/'.preg_quote($this->getTypesContent()).'/';
+        $pattern = '/'.preg_quote($this->getTypesContent(), '/').'/';
```

Comment:
you need to escape that `/` only when `getTypeContent` will return sth that may contain `/`. it it does not, not need for time wasting looking for `/` to escape it.
is it a case here? if so, provide utest for that